migrations/3_deploy_balancecopy.js

```diff
@@ -0,0 +1,40 @@
+var Token = artifacts.require("./LTOToken.sol");
+var BalanceCopier = artifacts.require("./BalanceCopier.sol");
+var config = require("../config.json");
+var tokenConfig = config.token;
+var balanceCopyConfig = config.balanceCopy;
+
+function convertDecimals(number, decimals) {
+  if (!decimals) {
+    decimals = tokenConfig.decimals;
+  }
+  return web3.toBigNumber(10).pow(decimals).mul(number);
+}
+
+module.exports = function(deployer, network, accounts) {
+  if (!balanceCopyConfig) return;
+
+  const oldToken = balanceCopyConfig.oldToken;
+  const maxSupply = convertDecimals(tokenConfig.maxSupply);
+  var tokenInstance = null;
+  var balanceCopierInstance = null;
+
+  return deployer.deploy(Token, tokenConfig.bridgeAddr, maxSupply)
+      .then(() => {
+        return deployer.deploy(BalanceCopier, oldToken, Token.address);
+      })
+      .then(() => {
+        return Token.deployed();
+      })
+      .then(instance => {
+        tokenInstance = instance;
+        return BalanceCopier.deployed();
+      })
+      .then(instance => {
+        balanceCopierInstance = instance;
+      })
+      .then(_ => tokenInstance.addPauser(BalanceCopier.address))
+      .then(_ => {
+        console.log(`Token: ${Token.address}`, `BalanceCopier: ${BalanceCopier.address}`);
+      });
```

Comment:
Don't you want to add the whole copying process inside the deployment of this script?

Reply:
We need to have a list of token holders. This must be generated using a (python) script that traverses through all Ethereum txs or events. We'll add that in a subsequent PR.